### PR TITLE
Add docs page for `torch.inf` and `torch.nan`

### LINKED ISSUE #131040: Create the pages which explain `torch.nan` and `torch.inf` in the doc
### 📚 The doc issue

I can use `torch.nan` and `torch.inf` as shown below but I cannot find the pages which explain `torch.nan` and `torch.inf` in [the doc](https://pytorch.org/docs/stable/index.html):

```python
import torch

torch.nan, torch.inf
# (nan, inf)
```

### Suggest a potential alternative/fix

So, the pages which explain `torch.nan` and `torch.inf` should be created in the doc.

cc @albanD

### PR DESCRIPTION
**Pull request by nluu175**
Fixes #131040

## Description
Add docs for `torch.inf` and `torch.nan`,

## Checklist
- [x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Comment by nluu175** [comment]
Hey @mikaylagawarecki , @ezyang , @janeyx99 could you please give me some feedbacks on this fix?

**Comment by ezyang** [comment]
sure. Waiting on doc render

**Comment by ezyang** [comment]
rip

**Comment by nluu175** [comment]
@ezyang I pushed some changes to address the previous errors. Could you please take a look?

**Comment by ezyang** [comment]
image

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
@svekars the way it's done here is a little crappy as these don't get hyperlinks but I don't think I care lol
